### PR TITLE
Code Flow controlled

### PR DESCRIPTION
**Pull request by Abhiramborige**
Please look into it .

### CODE REVIEW

**Comment by Abhiramborige** [comment]
Please look into it. 🙏🤝.
Sorry because I have contributed though you didn't assigned me for this. 
But I thought it would be great to help. 
I have made changes in the flow of code. A separate class for exception handling is created, which makes code to run smoothly even user inputs the option out of the given range. 

Here are screen shots:
image
image

File: quiz.py [hunk 1]
```diff
@@ -11,14 +19,19 @@ def start_quiz(questions):
...
-            answer = int(input("\nEnter your option number: ")); sleep(0.2)
-            if not 0<answer<5:
-                print(chalk.yellow("Please enter a choice between 1 and 4\n")); sleep(0.4)
-                continue
-        except ValueError:
-            print(chalk.cyan('Invalid Choice\n')); sleep(0.4)
-            continue
+        # flag for controlling the loop
+        flag = False
+        while(flag==False):
```
Comment (Sahil-k1509) [inline quiz.py:24]: I also added this while loop for taking valid input input but @prkhrv  said his quiz won't give second chances.

Comment (prkhrv) [reply quiz.py:24]: Yes, It's Not the Intention of a Quiz to give Second Chances.

Comment (Abhiramborige) [reply quiz.py:24]: Oh. Okay no worries.

**Pull request closed without merging**